pages/mentor-listing.tsx

```diff
@@ -0,0 +1,64 @@
+import "twin.macro";
+import { Box, HStack, Grid, Text, Link } from "@chakra-ui/react";
+import { shuffle, times } from "lodash";
+import { X } from "phosphor-react";
+
+import { FilterSidebar, MentorCard } from "components/pages/MentorListing";
+import { NavBar } from "components/modules/NavBar";
+import { useScreen } from "hooks";
+
+/* -------------------------------------------------------------------------- */
+/*                                  Mock Data                                 */
+/* -------------------------------------------------------------------------- */
+
+const avatarPaths = shuffle(times(16))
+    .slice(0, 12)
+    .map((i) => `/svg/avatar-${i}.svg`);
+
+const resultCount = 32;
+
+/* -------------------------------------------------------------------------- */
+/*                               Mentor Results                               */
+/* -------------------------------------------------------------------------- */
+
+function Results() {
+    return (
+        <Box tw="h-screen w-full" overflow="scroll">
+            <Grid tw="justify-start items-center p-8 gap-x-4 gap-y-1">
+                <Text tw="text-xl inline font-medium">Search Results For “programming”</Text>
+                <Link tw="text-sm flex" href="/mentor" gridColumn={2}>
+                    <X size={20} /> Clear All Filters
+                </Link>
+                <div tw="text-secondary uppercase">{resultCount} Results</div>
+            </Grid>
+            <Grid
+                tw="gap-4 w-full items-center"
+                p={[4, 8]}
+                justifyItems="center"
+                justifyContent="center"
+                templateColumns="repeat(auto-fit, minmax(375px, 1fr))">
+                {avatarPaths.map((image) => (
+                    <MentorCard key={image} image={image} />
+                ))}
+            </Grid>
+        </Box>
+    );
+}
+
+/* -------------------------------------------------------------------------- */
+/*                             Mentor Listing Page                            */
+/* -------------------------------------------------------------------------- */
+
+export default function MentorListing() {
+    const { min } = useScreen();
+    return (
+        // Setting the height=100vh and overflow=hidden get rid of the scrollbar
```

Comment:
You say here that it gets rid of the scrollbar, but when I look at the mentor listing page it still has the vertical scrollbar on the right-hand side. Do you mean the horizontal scrollbar?

Reply:
I guess this is for the grey background box. We have a vertical scrollbar for the card list, which is fine, but if you scroll down to the bottom without this setting, another scrollbar shows up. I think that's what this deals with

Reply:
This disables the scrollbar, but the lower parts of the bottommost cards are hidden😭

Reply:
Thinking of the solution for it, but I don't get it now. Maybe we can create a minor PR at some point. It's not that big issue.

Reply:
@ktsht What do you mean by "but the lower parts of the bottommost cards are hidden"?